update.sh

```diff
@@ -3,6 +3,16 @@
 KERNEL_VERSION_RPI1=3.18.0-trunk-rpi
 KERNEL_VERSION_RPI2=3.18.0-trunk-rpi2
 
+RASPBIAN_ARCHIVE_KEY_LOCATION="https://archive.raspbian.org/"
+RASPBIAN_ARCHIVE_KEY_NAME="raspbian.public.key"
+RASPBIAN_ARCHIVE_KEY_URL="${RASPBIAN_ARCHIVE_KEY_LOCATION}${RASPBIAN_ARCHIVE_KEY_NAME}"
```

Comment:
the location is a URL, which means you never really know (or should care) if it points to a directory or a file, it's a location... requiring that you have a trailing slash is quite error prone, IMHO